library/Backends/Michelson/src/Juvix/Backends/Michelson/DSL/InstructionsEff.hs

```diff
@@ -173,9 +177,9 @@ var symb = do
       pure (Env.Curr lamPartial)
     Just (VStack.Position (VStack.Usage usage _saved) index)
       | usage == one ->
-        Env.Expanded <$> moveToFront index
+        Env.Nop <$ moveToFront index
```

Comment:
I'm confused as to why this change is in this PR, was it just wrong? What does this have to do with the other changes?

Reply:
This makes 0 difference, in the past you've mentioned adding `Nop` as you found functions returning values and doing stack operations confusion. This would be another consistent place to add it

Reply:
Like with many changes, I found this while reasoning through the backend again.

Reply:
ah ah I see, yes this is better, thanks!